vyxal/helpers.py

```diff
@@ -759,6 +759,8 @@ def uncompress_dict(source: str) -> str:
                 if pos < len(vyxal.dictionary.small_dictionary):
                     ret += vyxal.dictionary.small_dictionary[pos]
                 temp_scc = ""
+                if char == " ":
+                    continue
```

Comment:
unrelated but why does this function use a deque? Why not just iterate over it?

Reply:
It's purely because deques are used in other places like the lexer and the parser.